lightning/src/ln/monitor_tests.rs

```diff
@@ -747,6 +753,238 @@ fn test_balances_on_local_commitment_htlcs() {
 	test_spendable_output(&nodes[0], &as_txn[1]);
 }
 
+#[test]
+fn test_no_preimage_inbound_htlc_balances() {
+	// Tests that MaybePreimageClaimableHTLCAwaitingTimeouts are generated for inbound HTLCs for
+	// which we do not have a preimage.
+	let chanmon_cfgs = create_chanmon_cfgs(2);
+	let node_cfgs = create_node_cfgs(2, &chanmon_cfgs);
+	let node_chanmgrs = create_node_chanmgrs(2, &node_cfgs, &[None, None]);
+	let mut nodes = create_network(2, &node_cfgs, &node_chanmgrs);
+
+	let (_, _, chan_id, funding_tx) = create_announced_chan_between_nodes_with_value(&nodes, 0, 1, 1_000_000, 500_000_000, InitFeatures::known(), InitFeatures::known());
+	let funding_outpoint = OutPoint { txid: funding_tx.txid(), index: 0 };
+
+	// Send two HTLCs, one from A to B, and one from B to C.
```

Comment:
Am I understanding this correctly? Because then the rest makes sense to me.
```suggestion
	// Send two HTLCs, one from A to B, and one from B to A.
```